src/pages/GoogleLogin.js

```diff
@@ -2,7 +2,7 @@ import React, { useRef, useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
 
-const GoogleLogin = () => {
+const GoogleLogin = props => {
   const history = useHistory();
```

Comment:
위의 `import { useHistory } from 'react-router-dom';` 이부분이 삭제됨에 따라 해당 부분은 다음과 같이 수정할 수 있습니다~

```suggestion
 const {history} = props;
```

Reply:
감사합니다 용민이님^^